src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
```

Comment:
The variable `result` is never reassigned, so it can be declared as a `const` instead of `let`. This is a code quality issue. Prefer `const` over `let` where possible, to avoid unintentional changes.